tests/test_other.py

```diff
@@ -7339,14 +7339,6 @@ def test_binaryen_invalid_method(self):
     proc.communicate()
     assert proc.returncode != 0
 
-  def test_binaryen_default_method(self):
```

Comment:
Yeah it seems weird to me to have a test-related print *only* really there for a test.